src/nnet3bin/nnet3-merge-egs.cc

```diff
@@ -30,7 +30,8 @@ namespace nnet3 {
 // or crashes if it is not there.
 int32 NumOutputIndexes(const NnetExample &eg) {
   for (size_t i = 0; i < eg.io.size(); i++)
-    if (eg.io[i].name == "output")
```

Comment:
You can't just change the implementation of functions without re-doing their documentation, like this!
Think about what would be the right way to this.